jobs/assets-subsystem/asset-sender/src/main/java/com/paladincloud/assetsender/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.paladincloud.assetsender;
```

Comment:
please take the asset-subsystem out of jobs and place it under root level as it might contain "things" other than jobs